docs/references/chrome-extension/create-clerk-client.mdx

```diff
@@ -0,0 +1,163 @@
+
+The following example assumes that you have followed the [Chrome Extension Quickstart](/docs/quickstarts/chrome-extension) and then the [Add React Router](/docs/references/chrome-extension/add-react-router) guide, but you can apply these concepts to your own application.
+
+<Steps>
+  ### Create your background service worker
+
+  1. In the `src/` directory, create the `background/` directory.
+  1. In the `background/` directory, create the `index.ts` file.
+  1. In the `index.ts` file, paste the following code to create an event listener that listens for messages from content scripts and calls a function that uses `createClerkClient()` to get a new token for the user.
+
+  ```typescript {{ filename: 'src/background/index.ts' }}
+  import { createClerkClient } from '@clerk/chrome-extension/background'
+
+  const publishableKey = process.env.PLASMO_PUBLIC_CLERK_PUBLISHABLE_KEY
+
+  if (!publishableKey) {
+    throw new Error('Please add the PLASMO_PUBLIC_CLERK_PUBLISHABLE_KEY to the .env.development file')
+  }
+
+  // Use `createClerkClient()` to create a new Clerk instance
+  // and get a fresh token for the user
+  async function getToken() {
+    const clerk = await createClerkClient({
+      publishableKey,
+    })
+    return await clerk.session?.getToken()
+  }
+
+  // Create a listener to listen for messages from content scripts
+  // It must return true, in order to keep the connection open and send a response later.
+  // NOTE: A runtime listener cannot be async.
+  chrome.runtime.onMessage.addListener((request, sender, sendResponse) => {
+    // This example sends the token back to the content script
+    // but you could also use the token to perform actions on behalf of the user
+    getToken().then((token) => sendResponse({ token }))
+    return true // REQUIRED: Indicates that the listener responds asynchronously.
+  })
+  ```
+
+  ### Create the tab with the content script
+
+  1. In the `src/` directory, create the `tabs/` directory.
+  1. In the `tabs/` directory, create the `background-worker-demo.html` file.
+  1. In the `background-worker-demo.html` file, paste the following code to create a basic HTML file that will house the React component for the content script.
+     ```html {{ filename: 'src/tabs/background-worker-demo.html' }}
+     <!doctype html>
+     <html>
+       <head>
+         <title>Clerk Background Worker Demo</title>
+         <meta charset="utf-8" />
+         <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+       </head>
+
+       <body></body>
+     </html>
+     ```
+  1. In the `tabs/` directory, create the `background-work-demo.tsx` file.
+  1. In the `background-work-demo.tsx` file, paste the following code to create a React component with a button that will trigger the background service worker to get the token. If the token is returned, it will be displayed on the page.
```

Comment:
```suggestion
  1. In the `background-worker-demo.tsx` file, paste the following code to create a React component with a button that will trigger the background service worker to get the token. If the token is returned, it will be displayed on the page.
```